### PR TITLE
Add InterMat-part2 

### PR DESCRIPTION
**Pull request by knc6**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/inermat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "intermat" %}
+{% set version = "2024.3.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/intermat/archive/refs/tags/v2024.3.24.tar.gz
```
Comment (ocefpaf) [inline recipes/inermat/meta.yaml:9]: We need that for the autoupdates.

```suggestion
  url: https://github.com/usnistgov/intermat/archive/refs/tags/v{{ version }}.tar.gz
```

File: recipes/inermat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  sha256: 747a041766d894123ac56308e4abaa58fb5f7088d59b7f96d8519093a2458cc2
+
+build:
+  noarch: python 
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+
+
```
Comment (ocefpaf) [inline recipes/inermat/meta.yaml:18]:
```suggestion
```

File: recipes/inermat/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  host:
+    - pip
+    - python >=3.8
+
+  run:
+    - jarvis-tools
+    - numpy >=1.19.5
+    - pytest
+    - python >=3.8
+
```
Comment (ocefpaf) [inline recipes/inermat/meta.yaml:29]:
```suggestion
```

File: recipes/inermat/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - intermat
+
+
+
```
Comment (ocefpaf) [inline recipes/inermat/meta.yaml:36]:
```suggestion
```

File: recipes/inermat/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - jarvis-tools
+    - numpy >=1.19.5
+    - pytest
+    - python >=3.8
+
+
+test:
+  imports:
+    - intermat
```
Comment (ocefpaf) [inline recipes/inermat/meta.yaml:33]:
```suggestion
    - intermat
  commands:
    - pip check
  requires:
    - pip
```

**Comment by knc6** [comment]
@ocefpaf ready for further review

**Pull request merged**